packages/indexer-cli/src/__tests__/references/indexer-rules.stdout

```diff
@@ -1,3 +1,11 @@
 Configure indexing rules
 
-
+  indexer rules stop (never)         Never index a deployment (and stop indexing it if necessary)     
+  indexer rules start (always)       Always index a deployment (and start indexing it if necessary)   
+  indexer rules set                  Set one or more indexing rules                                   
+  indexer rules prepare (offchain)   Offchain index a deployment (and start indexing it if necessary) 
```

Comment:
How does offchain indexing work if the allocation management mode is `manual`?

Reply:
In manual mode allocation management is turned off, but deployment management continues as before.  
As a result the behavior is very similar to before for `offchain` subgraphs.   i.e - `offchain` subgraphs will be deployed to the index nodes but allocations will not be created or managed for them.   The difference is if a subgraph with an active allocation is moved to `offchain` the agent will no longer close that allocation for you.